src/Symfony/Component/Intl/.gitignore

```diff
@@ -1,3 +1,9 @@
+Resources/data/zones/*.json
+!Resources/data/zones/de.json
+!Resources/data/zones/en.json
+!Resources/data/zones/fr.json
+!Resources/data/zones/meta.json
+!Resources/data/zones/nl.json
```

Comment:
For reference and my understanding, why this list?

Reply:
I think that's done to ease the review by embedding only a few locales in the diff, until the time we agree on the feature set (at which point the full data can be generated)